flujo/state/backends/sqlite_ops.py

```
@@ -218,6 +238,143 @@ async def list_states(self, status: Optional[str] = None) -> List[Dict[str, Any]
+        cutoff = datetime.now() - timedelta(hours=hours_back)
+        tasks = await self.list_background_tasks(parent_run_id=parent_run_id, status="failed")
+        filtered: List[Dict[str, Any]] = []
+        for task in tasks:
+            updated_at = task.get("updated_at")
+            try:
+                updated_dt = (
+                    datetime.fromisoformat(updated_at)
+                    if isinstance(updated_at, str)
+                    else updated_at
+                )
+            except Exception:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)